content/docs/command-reference/plots/index.md

```diff
@@ -260,3 +277,52 @@ file:///Users/usr/src/plots/classes.csv.html
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
 > predefined in DVC (found in `.dvc/plots/confusion.json`).
+
+### Example for the HTML template
+
+There may be times when you need to produce plots without Internet access, or
+want to customize the plots output to put some extra content, like banners or
+extra text. DVC allows to replace the HTML file that contains the final plots.
+
+Download the Vega libraries into the directory where you'll produce the
+`dvc plots`.
+
+```dvc
+$ wget https://cdn.jsdelivr.net/npm/vega@5.10.0 -O my_vega.js
+$ wget https://cdn.jsdelivr.net/npm/vega-lite@4.8.1 -O my_vega_lite.js
+$ wget https://cdn.jsdelivr.net/npm/vega-embed@6.5.1 -O my_vega_embed.js
+```
+
+Create the following HTML file with your text editor:
+
+```html
+<html>
+  <head>
+    <script src="my_vega.js" type="text/javascript"></script>
+    <script src="my_vega_lite.js" type="text/javascript"></script>
+    <script src="my_vega_embed.js" type="text/javascript"></script>
+  </head>
+  <body>
+    {plot_divs}
+  </body>
+</html>
+```
+
+Note that this is a standard HTML file with only `{plot_divs}` as a placeholder
+for DVC to inject plots. `<script>` tags in this file point to the local
+JavaScript libraries we downloaded above. Supposing this template was named
+`.dvc/page_template.html`, we can use it like this:
+
+```dvc
+$ dvc plots show --html-template .dvc/page_template.html
```

Comment:
````suggestion
Javascript libraries we downloaded above. We can use it like this:

```dvc
$ dvc plots show --html-template .dvc/plots/mypage.html
````

Reply:
For JavaScript spelling: https://dev.to/gypsydave5/how-to-spell-javascript-107d :))

Reply:
I didn't mean to change that spelling. Updated suggestion. I assume you're OK with the changes otherwise 🙂